starboard/linux/x64x11/gcc/6.3/platform_configuration/BUILD.gn

```diff
@@ -108,6 +112,24 @@ config("platform_configuration") {
       "//starboard/linux/x64x11/shared/platform_configuration:libraries",
       "//starboard/build/config/sabi",
       ":compiler_flags",
+      ":linker_flags",
     ]
+
+    # Workaround forcing a link to static library
+    # Resolves __wrap_eglSwapBuffers linking issues
+    libs = [ "starboard_egl_wrapped_functions" ]
```

Comment:
This is directly related to `"-Lstarboard/obj/starboard/linux/x64x11/shared",`, correct? If so, I think it would be best to group them in a config together

Reply:
Moved around .. and yet somehow this broke gcc-6.3

```
 error adding symbols: DSO missing from command line
```

I'll need to nudge this one more time a bit locally, the whole thing is very fragile.

Reply:
Can't practically avoid this `libs =` line here.

Reply:
What is the order of arguments necessary for this to work? Having a order-dependent libs in the middle of a config is fragile like you said, and is something we could improve by instead ensuring the order of configs is correct or otherwise.